warehouse/query-core/src/main/java/datawave/query/jexl/DatawaveJexlEngine.java

```diff
@@ -37,4 +38,10 @@ protected Interpreter createInterpreter(JexlContext context, boolean strictFlag,
     public ASTJexlScript parse(CharSequence expression) {
         return super.parse(expression, null, null);
     }
+    
+    @Override
+    protected ASTJexlScript parse(CharSequence expression, JexlInfo info, Scope frame) {
+        expression = expression.toString().replace("\\\\", "\\\\\\\\");
```

Comment:
replacing 4 slashes with 8 seems weird.  I think instead we should be replacing every 2 slashes with 4.  You have been focused on the double backslash thing but don't we have the same issues with the single backslash?

Reply:
Check the latest commit.  I decided to move away from fixing the slashes here.